vm/src/import.rs

```diff
@@ -144,3 +144,35 @@ fn find_source(vm: &VirtualMachine, current_path: PathBuf, name: &str) -> Result
         None => Err(format!("No module named '{}'", name)),
     }
 }
+
+pub fn remove_importlib_frames(vm: &VirtualMachine, exc: &PyObjectRef) -> PyObjectRef {
+    let always_trim = objtype::isinstance(exc, &vm.ctx.exceptions.import_error);
+
+    if let Ok(tb) = vm.get_attribute(exc.clone(), "__traceback__") {
+        if objtype::isinstance(&tb, &vm.ctx.list_type()) {
+            let mut tb_entries = objsequence::get_elements_list(&tb).to_vec();
+            tb_entries.reverse();
```

Comment:
It looks like the list is reversed twice. This might not be required.

Reply:
Fixed.